apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+  siteId: string
+  folderId: string
+}
+
+// export const createPageSchema = z.object({
+//   pageName: z.string(),
+//   pageTitle: z.string(),
+//   // TODO: add the actual layouts in here
+//   layout: z.enum(PAGE_LAYOUTS).default('content'),
+//   siteId: z.string(),
+//   // NOTE: implies that top level pages are allowed
+//   folderId: z.string().nullable(),
+// })
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutTypename: LayoutType
+  layoutDescription: string
+  imageSource: string
+  previewJson: JSON
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.png',
+    previewJson: contentLayoutPreview,
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.png',
+    previewJson: articleLayoutPreview,
+  },
+]
+
+// TODO: Make this headless by getting the LAYOUT_DATA from the schema. Find somewhere in the schema for layoutDescription & image(fetch this too or generate it)
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  // need state here to keep track of selected layout
+  const [selectedLayout, setSelectedLayout] = useState(LAYOUT_DATA[0]!)
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100%">
+        <VStack p="1.5rem 2rem" w="full" gap="0.5rem" alignItems="flex-start">
+          <Button variant="link">
+            <Icon as={BiLeftArrowAlt} />
+            Back to all pages
+          </Button>
+          <HStack p="0" justifyContent="space-between" w="100%">
+            <Text textStyle="h4" color="base.content.default">
+              {`Choose a layout for "`}
+              {props.pageName}
+              {`"`}
+            </Text>
+            <Button size="xs">
+              Start with this layout
+              <BiRightArrowAlt fontSize="1.5rem" />
+            </Button>
+          </HStack>
+        </VStack>
+        <Grid
+          w="100%"
+          h="calc(100% - 7rem)"
+          templateColumns="repeat(4, 1fr)"
+          gap={0}
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((layoutEntry) => {
+                return (
+                  <VStack
+                    key={layoutEntry.layoutTypename}
+                    p="0"
+                    gap="0"
+                    cursor="pointer"
+                    onClick={() => {
+                      setSelectedLayout(layoutEntry)
+                    }}
+                    role="group"
+                  >
```

Comment:
this should be a `button` tag instead for semantic purposes. admittedly not important for non-public screens but good to be mindful. Suggest making it a button (but not the Button component) haha.

can be replaced like so:
```suggestion
                  <VStack
                    key={layoutEntry.layoutTypename}
                    p="0"
                    gap="0"
                    // cursor="pointer" // Don't need cursor anymore since button inherently has that style
                    onClick={() => {
                      setSelectedLayout(layoutEntry)
                    }}
                    role="group"
                    as="button"   // This is the change~
                  >
```